### ISSUE TITLE
Unable to delete configuration when name contains "/".

### ISSUE THREAD

**Issue by miha-anzur** [dynatrace-oss/dynatrace-monitoring-as-code#255]
Hello 

Unable to delete configuration when configuration name contains "/".

delete file content:

```
delete:
- 'anomaly-detection-metrics/EC_WCM_LOCAL_DISK_Critical_ers-csp_ec/prod'
```

Result:
///////////////////////////////////////////////////////////////////
ERROR deletion failed: config anomaly-detection-metrics/EC_WCM_LOCAL_DISK_Critical_ers-csp_/ec contains more than one '/' delimiter
///////////////////////////////////////////////////////////////////

Encoding "/" characters does not help

```
delete:
- "anomaly-detection-metrics/EC_WCM_LOCAL_DISK_Critical_ers-csp_\u002Fec"
```

Escaping with "\\" neither.

**Comment by kristofre** [comment]
@patrickpichler would this be something that is resolved when we are going for the new yaml syntax?

**Comment by patrickpichler** [comment]
I didn't plan (at least yet) to rework the delete.yaml. So sadly no.

**Comment by patrickpichler** [comment]
@miha-anzur on what OS are you?

**Comment by patrickpichler** [comment]
Actually I just noticed, do you really have a `/` in your config id name?

**Comment by miha-anzur** [comment]
Hello

We are using linux.

Linux filesystems are containing "/".
When we are configuring rules for specific filesystem on specific host we have to have an option to see what is the rule about.
The only place is the name.

**Comment by patrickpichler** [comment]
As of right now, this will cause issues. I would recommend switching away 
from names containing `/`. Maybe replace it with a dash `-`. 

@dynatrace-oss/mac-maintainers we should probably prohibit it on a config 
level that a config id can contain a `/`, or what do you think?

**Comment by tatomir146** [comment]
@patrickpichler we should consider moving delete.yaml also to the new configuration where using any supported characters would be possible. Delete.yaml anyways need a rework so it can be executed on per tenant/config basis.

**Comment by patrickpichler** [comment]
As @kristofdynatrace pointed out in #218 it might beneficial to redesign the whole
`delete.yaml` feature.

**Comment by patrickpichler** [comment]
This should be fixed with #351

**Comment by patrickpichler** [comment]
Will be fixed with release of version 2.0
